txclib/project.py

```diff
@@ -394,7 +394,7 @@ def _slug_with_branch(self, resource_slug, branch):
     def pull(self, languages=None, resources=None, overwrite=True,
              fetchall=False, fetchsource=False, force=False, skip=False,
              minimum_perc=0, mode=None, pseudo=False, xliff=False, branch=None,
-             parallel=False, no_interactive=False):
+             parallel=False, no_interactive=False, use_git_timestamps=False):
```

Comment:
`use_git_timestamps` argument should be passed to `_languages_to_pull()` method invocation inside `pull` body.